.github/workflows/cpp.yml

```diff
@@ -113,6 +113,23 @@ jobs:
           cd cpp/examples/minimal_build
           docker-compose run --rm minimal
 
+  cmake-checks:
+    name: CMake checks
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout Arrow
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run
+        run: |
+          cd cpp
+          cmake --list-presets
```

Comment:
@pitrou could we call this from one of the existing C++ builds so we don't need to run an additional build? Perhaps from build-example or the macos one.

Reply:
If you prefer, though the additional build takes 10 seconds.